### PR TITLE
AV-34334 #comment SDK changes to support changing django session cook…

### PR DESCRIPTION
**Pull request by skdas**
…ie name

### CODE REVIEW

**Review by gcox1 — APPROVED** [review]

**Comment by grastogi23** [comment]
@skdas can you pl. check if the pytest runs successfully for this code.

**Comment by skdas** [comment]
Avitest went through fine
```
==================================================================== test session starts =====================================================================
platform linux2 -- Python 2.7.6, pytest-3.3.1, py-1.5.2, pluggy-0.6.0 -- /usr/bin/python
cachedir: ../../../../../.cache
metadata: {'Python': '2.7.6', 'Platform': 'Linux-4.4.0-87-generic-x86_64-with-Ubuntu-14.04-trusty', 'Packages': {'py': '1.5.2', 'pytest': '3.3.1', 'pluggy': '0.6.0'}, 'Plugins': {'html': '1.16.0', 'metadata': '1.5.1'}}
rootdir: /home/sambit, inifile:
plugins: metadata-1.5.1, html-1.16.0
collected 18 items



test_waf_crud.py::Test_Waf_Crud::test_setup PASSED                                                                                                     [  5%]
test_waf_crud.py::Test_Waf_Crud::test_enforcement_mode[virtualservice-1-/wafblah-410-100001-p1g1] PASSED                                               [ 11%]
test_waf_crud.py::Test_Waf_Crud::test_group_enable_disable[virtualservice-1-/wafblah-410] PASSED                                                       [ 16%]
test_waf_crud.py::Test_Waf_Crud::test_group_change_group_name PASSED                                                                                   [ 22%]
test_waf_crud.py::Test_Waf_Crud::test_all_group_disable_enable[virtualservice-1-/wafblah-410] PASSED                                                   [ 27%]
test_waf_crud.py::Test_Waf_Crud::test_whitelisting_remote_ip[virtualservice-1-/wafblah-410] PASSED                                                     [ 33%]
test_waf_crud.py::Test_Waf_Crud::test_whitelisting_uri[virtualservice-1-/wafblah-410] PASSED                                                           [ 38%]
test_waf_crud.py::Test_Waf_Crud::test_group_name_change[virtualservice-1-/wafblah-410] PASSED                                                          [ 44%]
test_waf_crud.py::Test_Waf_Crud::test_group_xclusion_change[virtualservice-1-/wafblah-410] PASSED                                                      [ 50%]
test_waf_crud.py::Test_Waf_Crud::test_profile_default_action_change[virtualservice-1-/wafblah-403] PASSED                                              [ 55%]
test_waf_crud.py::Test_Waf_Crud::test_rule_enable_disable[virtualservice-1-/wafblah-410] PASSED                                                        [ 61%]
test_waf_crud.py::Test_Waf_Crud::test_all_rule_disable_enable[virtualservice-1-/wafblah-410] PASSED                                                    [ 66%]
test_waf_crud.py::Test_Waf_Crud::test_rule_xclusion_change[virtualservice-1-/wafblah-410] PASSED                                                       [ 72%]
test_waf_crud.py::Test_Waf_Crud::test_rule_add_del_rules[virtualservice-1-/wafblah-410] PASSED                                                         [ 77%]
test_waf_crud.py::Test_Waf_Crud::test_invalid_modsec_rule PASSED                                                                                       [ 83%]
test_waf_crud.py::Test_Waf_Crud::test_that_system_objects_cant_be_deleted[wafpolicy-System-WAF-Policy] PASSED                                          [ 88%]
test_waf_crud.py::Test_Waf_Crud::test_that_system_objects_cant_be_deleted[wafprofile-System-WAF-Profile] PASSED                                        [ 94%]
test_waf_crud.py::Test_Waf_Crud::test_delete PASSED                                                                                                    [100%]
Time taken to generate robot xml file 0.13 seconds
Rebot framework is generating html file...

--- rebot xml from /home/sambit/log.xml to /home/sambit/log.html ---
Log:     /home/sambit/log.html
Report:  /home/sambit/log_report.html
Time taken for rebot to generate html file 1.14 seconds


---------------------------------------------------------- generated xml file: /home/sambit/log.xml ----------------------------------------------------------
================================================================ 18 passed in 165.77 seconds =================================================================
```

**Comment by skdas** [comment]
@grastogi23 @gcox1 - avitests went through fine. I have attached the results here. Can you please merge both the PRs

File: python/avi/sdk/avi_api.py [hunk 1]
```diff
@@ -227,6 +227,7 @@ def __init__(self, controller_ip=None, username=None, password=None,
         self.verify = verify
         self.retry_conxn_errors = retry_conxn_errors
         self.remote_api_version = {}
+        self.session_cookie_name = ''
```
Comment (grastogi23) [inline python/avi/sdk/avi_api.py:230]: should we initialize it to sessionid?

**Pull request merged**